digdag-standards/src/test/resources/io/digdag/standards/operator/for_each/basic_expected.yml

```diff
@@ -1,6 +1,6 @@
-+for-a=1:
++for-0=a=0=1:
```

Comment:
`@` does seem to be allowed in task names. https://github.com/treasure-data/digdag/blob/master/digdag-core/src/main/java/io/digdag/core/repository/ModelValidator.java#L26

Is `+for-0@a=1@2&1@b=1@4:` less confusing than `+for-0=a=1=2&1=b=1=4:` ?